cli/asc.js

```diff
@@ -21,6 +21,7 @@ const colorsUtil = require("./util/colors");
 const optionsUtil = require("./util/options");
 const mkdirp = require("./util/mkdirp");
 const EOL = process.platform === "win32" ? "\r\n" : "\n";
+const sep = process.platform === "win32" ? path.win32.sep : path.sep;
```

Comment:
Perhaps `const SEP = process.platform === "win32" ? "\\" : "/";`,to match the EOL constant.